compiler/one-cmds/one-init

```diff
@@ -0,0 +1,262 @@
+        help='show program\'s version number and exit')
+
+    # model path
+    parser.add_argument(
+        '--model_path',
+        required=False,
+        type=str, default='',
+        help='the model path to be used for creating config file')
+
+    return parser
+
+
+def _parse_arg(parser):
+    # NOTE: Trick for parsing args
+    # To handle both `one-init` and `one-init ${model}`,
+    # `one-init ${model}` -> `one-init --model_path ${model}`
+    argv = copy.deepcopy(sys.argv[1:])
+    if len(argv) > 0 and not argv[0].startswith('--'):
+        argv.insert(0, '--model_path')
+    args = parser.parse_args(argv)
+
+    # print version
+    if args.version:
+        _utils._print_version_and_exit(__file__)
+
+    return args
+
+
+def _create_init_cfg(args):
+    template_path = args.template_path
+    dir_path = os.getcwd()
+    cfg_path = os.path.join(dir_path, 'init.cfg')
+
+    if os.path.exists(cfg_path) is True:
+        raise FileExistsError(cfg_path + ' file already exists')
+
+    # create init.cfg file by coping
+    try:
+        shutil.copyfile(template_path, cfg_path)
+    except (shutil.Error, OSError):
+        raise shutil.Error('Unable to copyfile: %s -> %s' % template_path, cfg_path)
+
+
+def _create_type_cfg(args):
+    template_path = args.template_path
+    model_path = args.model_path
+
+    dir_path = os.getcwd()
+    model_name, ext = os.path.splitext(os.path.basename(model_path))
+    ext = ext[1:] # .ext -> ext
+    model_type = _get_model_type(ext)
+
+    cfg_name = model_name + '.cfg'
+    cfg_path = os.path.join(dir_path, cfg_name)
+
+    if os.path.exists(cfg_path) is True:
+        raise FileExistsError(cfg_path + ' file already exists')
+
+    # create cfg file by coping
+    try:
+        shutil.copyfile(template_path, cfg_path)
+    except (shutil.Error, OSError):
+        raise shutil.Error('Unable to copyfile: %s -> %s' % template_path, cfg_path)
+
+    # modify cfg file according to each model type
+    dir_path = os.path.dirname(model_path)
+    import_type = f'one-import-{model_type}' # f string format >= py3.6
+
+    # assume that the template file is validated
+    cfg_parser = TemplateConfigParser()
+    cfg_parser.read(cfg_path)
+
+    section = '[onecc]'
+    cfg_parser.set(section, import_type, 'True')
+    cfg_parser.set(section, 'one-optimize', 'True')
+    cfg_parser.set(section, 'one-quantize', 'True')
+
+    section = f'[{import_type}]'
+    input_path = os.path.join(dir_path, model_name + '.' + ext)
+    output_path = os.path.join(dir_path, model_name + '.circle')
+    cfg_parser.set(section, 'input_path', input_path)
+    cfg_parser.set(section, 'output_path', output_path)
+
+    section = '[one-optimize]'
+    input_path = os.path.join(dir_path, model_name + '.circle')
+    output_path = os.path.join(dir_path, model_name + '.opt.circle')
+    cfg_parser.set(section, 'input_path', input_path)
+    cfg_parser.set(section, 'output_path', output_path)
+
+    section = '[one-quantize]'
```

Comment:
I make `one-quantize` as default. Is it good?

Reply:
@YongseopKim , how about reference the `dotnet` CLI style?
We could try using the backend type as the template name. For example `onecc init onert` or `onecc init tv2`, etc. Depending on the type of each template, we will be able to take the default value of whether to enable/disable each step.

```
sjlee@jupiter:~ $ dotnet new --list
These templates matched your input:

Template Name                                 Short Name      Language    Tags
--------------------------------------------  --------------  ----------  --------------------------
ASP.NET Core Empty                            web             [C#],F#     Web/Empty
ASP.NET Core gRPC Service                     grpc            [C#]        Web/gRPC
ASP.NET Core Web API                          webapi          [C#],F#     Web/WebAPI
ASP.NET Core Web App                          webapp,razor    [C#]        Web/MVC/Razor Pages
ASP.NET Core Web App (Model-View-Controller)  mvc             [C#],F#     Web/MVC
ASP.NET Core with Angular                     angular         [C#]        Web/MVC/SPA
ASP.NET Core with React.js                    react           [C#]        Web/MVC/SPA
Blazor Server App                             blazorserver    [C#]        Web/Blazor
Blazor WebAssembly App                        blazorwasm      [C#]        Web/Blazor/WebAssembly/PWA
Class Library                                 classlib        [C#],F#,VB  Common/Library
Console App                                   console         [C#],F#,VB  Common/Console
dotnet gitignore file                         gitignore                   Config
Dotnet local tool manifest file               tool-manifest               Config
EditorConfig file                             editorconfig                Config
global.json file                              globaljson                  Config
MSTest Test Project                           mstest          [C#],F#,VB  Test/MSTest
MVC ViewImports                               viewimports     [C#]        Web/ASP.NET
MVC ViewStart                                 viewstart       [C#]        Web/ASP.NET
NuGet Config                                  nugetconfig                 Config
NUnit 3 Test Item                             nunit-test      [C#],F#,VB  Test/NUnit
NUnit 3 Test Project                          nunit           [C#],F#,VB  Test/NUnit
Protocol Buffer File                          proto                       Web/gRPC
Razor Class Library                           razorclasslib   [C#]        Web/Razor/Library
Razor Component                               razorcomponent  [C#]        Web/ASP.NET
Razor Page                                    page            [C#]        Web/ASP.NET
Solution File                                 sln                         Solution
Web Config                                    webconfig                   Config
Worker Service                                worker          [C#],F#     Common/Worker/Web
xUnit Test Project                            xunit           [C#],F#,VB  Test/xUnit
```

Reply:
This will also help support the https://github.com/Samsung/ONE/issues/9260 feature. :)

Reply:
I made an issue https://github.com/Samsung/ONE/issues/9264